src/ExadelMentorship.Auth/IdentityConfiguration.cs

```diff
@@ -21,7 +19,8 @@ public class IdentityConfiguration
         public static IEnumerable<ApiScope> Scopes => new[]
         {
             new ApiScope("WebApi.read"),
-            new ApiScope("role")
+            new ApiScope("role"),
```

Comment:
Probably not needed. Usually, we use "role"s as claims, not scopes.

Reply:
I tested many times, it's not working without. Even in your provided example it exist. 
<img width="716" alt="image" src="https://user-images.githubusercontent.com/59729680/178141771-c8675e9d-37ac-4ea1-8f4b-926444c0cca9.png">